scipy/sparse/base.py

```diff
@@ -100,7 +101,26 @@ def get_shape(self):
 
     shape = property(fget=get_shape, fset=set_shape)
 
-    def reshape(self, shape):
+    def reshape(self, shape, order='C'):
+        """
+        Gives a new shape to a sparse matrix without changing its data.
+
+        Parameters
+        ----------
+        shape : length-2 tuple of ints
+            The new shape should be compatible with the original shape.
+        order : 'C', optional
+            This argument is in the signature *solely* for NumPy
+            compatibility reasons.
```

Comment:
This means don't pass in anything for `order` and leave it as the default value `'C'` because this argument is only in the signature for compatibility reasons.